stdlib/bigarray.mli

```diff
@@ -233,10 +233,10 @@ type fortran_layout = Fortran_layout_typ (**)
    and [(x+1, y)] are adjacent in memory.
 
    Each layout style is identified at the type level by the
-   phantom types {!Bigarray.c_layout} and {!Bigarray.fortran_layout}
+   phantom types {!Bigarray.type-c_layout} and {!Bigarray.type-fortran_layout}
```

Comment:
Same as above: `{!type:Bigarray.c_layout}` and `{!type:Bigarray.fortran_layout}`.